frontend/src/libs/components/sidebar/styles.module.scss

```diff
@@ -18,10 +18,13 @@
 }
 
 .sidebar {
+  position: sticky;
+  top: 0;
   display: flex;
   flex-direction: column;
   gap: 262px;
```

Comment:
@Liza-Veis Maybe it's also an option to make appropriate icons centering in this issue?

![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92298163/06fa0fe3-7c1b-408d-a51b-c4c3320c93d5)

Reply:
what do you mean by icons centering? if you mean that they should be at the center of the screen, from design it looks like they are not centered vertically

Reply:
Okay, I realized it. It may cause lack of responsiveness, but if it's needed this way, no more comments)

Reply:
we will think about it a bit later, let's leave it for now